colorIds.gs

```diff
@@ -0,0 +1,14 @@
+var colorIds = [
+  "unknown",
```

Comment:
Lets make this "default" rather than "unknown"

Reply:
For showing the user the available options i would rather link to the official doc [https://developers.google.com/apps-script/reference/calendar/event-color](https://developers.google.com/apps-script/reference/calendar/event-color)

No need for an additional file/manually updated array

Reply:
Got it, added a reference to it and removed the custom array.